source/javascripts/app/builds/app.js

```diff
@@ -194,12 +194,12 @@ App.Project.reopenClass({
       baseFileName: 'ember',
       projectFilter: [ /ember\./, /ember-template-compiler/ ],
       projectRepo: 'emberjs/ember.js',
-      initialVersion: "1.13.0",
```

Comment:
Ah I understand the schema now - I was basing my changes on the previous commit on the file, which was a minor release, not a point release. I'll fix this and do another PR.